plugins/modules/info.py

```diff
@@ -2889,7 +3082,8 @@ def perform_module_operation(self):
             ClusterIdentity=cluster_identity,
             ClusterOwner=cluster_owner,
             SnmpSettings=snmp_settings,
-            ServerCertificate=server_certificate
+            ServerCertificate=server_certificate,
+            SupportAssistSettings=support_assist_settings
```

Comment:
we had a comments from anupam on the naming conventions, should we name this as support_assist_settings instead ?